app/ClientSrc/vue/coa/ApplicationTypeSelect.vue

```diff
@@ -0,0 +1,344 @@
+<template>
+    <div class="application-type-select">
+        <!-- button to launch the modal dialog -->
+        <button
+            :disabled="disabled"
+            @click="showModal"
+            type="button"
+            class="btn btn-radio btn-radio--secondary"
+        >
+            Choose Application Type(s)
```

Comment:
I think there shouldn't be a hover color for this button when it's disabled

Reply:
I added the "disabled" class to prevent this, so now it should behave the same as the other buttons in that form. Incidentally this is styled as a radio button, just because that's the style that was used in the mockup!